manager/controllers/default/security/login.class.php

```diff
@@ -33,39 +36,42 @@ public function checkPermissions() {
 
     /**
      * Register custom CSS/JS for the page
+     *
      * @return void
      */
-    public function loadCustomCssJs() {}
+    public function loadCustomCssJs() {
+    }
 
     /**
      * Custom logic code here for setting placeholders, etc
+     *
      * @param array $scriptProperties
+     *
      * @return void
+     * @throws Exception
      */
     public function process(array $scriptProperties = array()) {
         $this->handleForgotLoginHash();
+        $this->handleMagicLoginLink();
         $this->preserveReturnUrl();
 
         if (!empty($this->scriptProperties)) {
             $this->handlePost();
         }
 
         /* invoke OnManagerLoginFormPrerender event */
-        $eventInfo= $this->modx->invokeEvent('OnManagerLoginFormPrerender');
-        $eventInfo= is_array($eventInfo) ? implode("\n", $eventInfo) : (string) $eventInfo;
+        $eventInfo = $this->modx->invokeEvent('OnManagerLoginFormPrerender');
+        $eventInfo = is_array($eventInfo) ? implode("\n", $eventInfo) : (string)$eventInfo;
         $this->setPlaceholder('onManagerLoginFormPrerender', $eventInfo);
 
         $hour = date('H') + (int)$this->modx->getOption('server_offset');
         if ($hour > 18) {
             $greeting = $this->modx->lexicon('login_greeting_evening');
-        }
-        elseif ($hour > 12) {
+        } else if ($hour > 12) {
```

Comment:
Why did you add these code style changes? Seems unnecessary (and not an improvement, but I know code styles are very subjective)

Reply:
@wuuti Please use `elseif` instead of `else if`, since you could write:

```
        if ($a) {
            // code
        } elseif ($b) {
            // code
        }
```

instead of:

```
        if ($a) {
            // code
        } else {
            if ($b) {
                // code
            }
        }
```

If we use an autoformatter (with some strict rules) in 3.x, the second code would be generated.

Reply:
I reverted any code style change (accidentally autoformatted by my IDE).